charts/metering-ansible-operator/templates/crds/meteringconfig.crd.yaml

```diff
@@ -94,23 +94,40 @@ spec:
                       - container
                       - secretName
                       properties:
+                        createSecret:
+                          type: boolean
                         container:
                           type: string
+                          minLength: 1
                         secretName:
                           type: string
+                          minLength: 1
+                        storageAccountName:
+                          type: string
+                          minLength: 1
+                        secretAccessKey:
+                          type: string
+                          minLength: 1
                         rootDirectory:
                           type: string
+                          minLength: 1
```

Comment:
specifying `minLength: 1` on a non-required property, just ensures that the `spec.storage.hive.azure.rootDirectory` key is non-empty, when it's specified in a `MeteringConfig` CR.

Reply:
Yep, awesome